controller/src/test/java/io/pravega/controller/store/stream/StreamMetadataStoreTest.java

```diff
@@ -460,7 +463,93 @@ public void scaleTest() throws Exception {
         latch.complete(null);
 
         AssertExtensions.assertThrows("", resp, e -> Exceptions.unwrap(e) instanceof ScaleOperationExceptions.ScaleConflictException);
+        // endregion
+    }
+
+    @Test
+    public void concurrentStartScaleTest() throws Exception {
+        final String scope = "ScopeScale";
+        final String stream = "StreamScale";
+        final ScalingPolicy policy = ScalingPolicy.fixed(2);
+        final StreamConfiguration configuration = StreamConfiguration.builder().scope(scope).streamName(stream).scalingPolicy(policy).build();
+
+        long start = System.currentTimeMillis();
+        store.createScope(scope).get();
+
+        store.createStream(scope, stream, configuration, start, null, executor).get();
+        store.setState(scope, stream, State.ACTIVE, null, executor).get();
+
+        // region concurrent start scale requests where one request starts and completes as the other is waiting on StartScale
```

Comment:
I'm not sure why this comment is starting with region. I think you are trying to both start a region and write a comment down.